search/search_index_values.hpp

```diff
@@ -25,43 +25,69 @@
 // A wrapper around feature index.
 struct FeatureIndexValue
 {
-  FeatureIndexValue() : m_featureId(0) {}
+  FeatureIndexValue() : m_id(0) {}
 
-  FeatureIndexValue(uint64_t featureId) : m_featureId(featureId) {}
+  explicit FeatureIndexValue(uint64_t id) : m_id(id) {}
 
-  bool operator<(FeatureIndexValue const & o) const { return m_featureId < o.m_featureId; }
+  bool operator<(FeatureIndexValue const & o) const { return m_id < o.m_id; }
 
-  bool operator==(FeatureIndexValue const & o) const { return m_featureId == o.m_featureId; }
+  bool operator==(FeatureIndexValue const & o) const { return m_id == o.m_id; }
 
-  void Swap(FeatureIndexValue & o) { swap(m_featureId, o.m_featureId); }
+  void Swap(FeatureIndexValue & o) { swap(m_id, o.m_id); }
 
-  uint64_t m_featureId;
+  uint64_t m_id;
```

Comment:
Please add a comment that this "id" is specifically a "featureId".
The situation with various ids is already bad, no need to make it worse.

Reply:
I reverted this changes.